app/views/shared/navigation/second_nav/_manage_school_group_menu.html.erb

```diff
@@ -13,6 +13,11 @@
                 school_group_energy_tariffs_path(school_group), class: 'dropdown-item' %>
     <%= link_to t('manage_school_menu.digital_signage'),
                 school_group_digital_signage_index_path(school_group), class: 'dropdown-item' %>
+    <% if current_user.admin? || (current_user.group_admin? &&
```

Comment:
Minor but you could also just enable the feature for admins in the deployment task and then just check of the feature is enabled for the current user.